Helm can use revision tags (#10112)
istio/istio.io: content/en/docs/setup/upgrade/canary/index.md

## Stable revision labels (experimental)

{{< tip >}}
If you're using Helm, refer to the [Helm upgrade documentation](/docs/setup/upgrade/helm).
{{</ tip >}}

{{< boilerplate revision-tags-preamble >}}

### Usage

{{< boilerplate revision-tags-usage >}}

{{< text bash >}}
$ istioctl tag set prod-stable --revision 1-9-5
$ istioctl tag set prod-canary --revision 1-10-0
{{< /text >}}

{{< boilerplate revision-tags-middle >}}

{{< text bash >}}
$ istioctl tag set prod-stable --revision 1-10-0
{{< /text >}}

{{< boilerplate revision-tags-prologue >}}

### Default tag

{{< boilerplate revision-tags-default-intro >}}

{{< text bash >}}
$ istioctl tag set default --revision 1-10-0
{{< /text >}}

> {{< boilerplate revision-tags-default-intro >}}

Reviewer: This is the same boilerplate that is mentioned on line 148, and the boilerplate ends in:
```
To make a revision `1-10-0` the default, run:
```
Line 148 does have the text box after the boilerplate, but it's not added here.

Reviewer: Maybe this one should be outro.

Author: Fixed in both versions

## Stable revision labels (experimental)

{{< tip >}}

> If using Helm and you would prefer to use Helm over `istioctl` please refer to the [Helm upgrade documentation](/docs/setup/upgrade/helm).

Reviewer 2: ```suggestion
If you're using Helm, refer to the [Helm upgrade documentation](/docs/setup/upgrade/helm).
```
